netty/src/main/java/io/grpc/transport/netty/NettyClientHandler.java

```diff
@@ -182,7 +193,9 @@ WriteQueue getWriteQueue() {
    */
   void returnProcessedBytes(Http2Stream stream, int bytes) {
     try {
-      decoder().flowController().consumeBytes(ctx, stream, bytes);
+      decoder().flowController().consumeBytes(stream, bytes);
+      decoder().flowController().writeWindowUpdates(ctx);
```

Comment:
so for each stream that returns processed bytes, we will iterate across all streams and write window updates if appropriate (O(n^2))?